ops/jujuversion.py

```diff
@@ -123,3 +123,9 @@ def has_secrets(self) -> bool:
         # * In 3.0.3, a bug with observer labels was fixed (juju/juju#14916)
         # TODO(benhoyt): update to 3.0.3+ once shipped (for juju/juju#14916 fix)
         return (self.major, self.minor, self.patch) >= (3, 0, 2)
```

Comment:
I'm guessing you want to set this to 3,0,3 given the associated comment. (Not necessarily in this PR, but looks like a TODO that we forgot to follow up on)

Reply:
Fixed in https://github.com/canonical/operator/pull/967. Is it kosher to remove the fallback now that 3.0.3 has been out for a long time? `get_secret` will fail if some people are still running 3.0.2, but that seems unlikely, and if it happens, we can just tell people "please upgrade to 3.0.3 or a later 3.x version". Do you know what we normally do in cases like this?

Reply:
> I would mark it clearly in the release notes associated with that version of python-libjuju.

Thanks. I presume you mean "that version of *ops*"?